python/cuml/cuml/internals/api_decorators.py

```diff
@@ -149,6 +159,8 @@ def wrapper(*args, **kwargs):
 
                     if input_arg_:
                         input_val = _get_value(args, kwargs, *input_arg_)
+                        if GlobalSettings().accelerator_active:
+                            input_val = support_array_like(input_val)
```

Comment:
It looks like we should have an additional test here to ensure that there are no regression regarding the ability for an estimator to ingest an array-like input.

Reply:
Added unit tests in https://github.com/rapidsai/cuml/commit/4803ce42910baddd7f9ab1754f3ef2b239610f7e#diff-c07f109cbff928c426246968e5e3b4f8ccb5a027dc97b7d5a141168f8799f950 .